src/app/query/common/alex-sdk/alex-sdk.hooks.ts

```diff
@@ -0,0 +1,36 @@
+import { useCallback, useState } from 'react';
+
+import { AlexSDK, type Currency } from 'alex-sdk';
+import BigNumber from 'bignumber.js';
+
+import { logger } from '@shared/logger';
+import { createMoney } from '@shared/models/money.model';
+import { isDefined } from '@shared/utils';
+
+import { convertAmountToFractionalUnit } from '@app/common/money/calculate-money';
+import { pullContractIdFromIdentity } from '@app/common/utils';
+
+import { useAlexSdkLatestPricesQuery } from './latest-prices.query';
+import { useAlexSdkSwappableCurrencyQuery } from './swappable-currency.query';
+
+export function useAlexSdKCurrencyPriceAsMoney() {
+  const alexSDK = useState(() => new AlexSDK())[0];
```

Comment:
Don't we use this already elsewhere? Can we have a single hook place somewhere to init the hook? To avoid performance issues, can't we render it outside react?

Reply:
This is prob true and will check. The Alex team implemented it this way, so I kept it.